.codesandbox/ci.json

```diff
@@ -1,5 +1,6 @@
 {
+  "packages": ["packages/waku"],
   "buildCommand": "compile",
   "sandboxes": ["vanilla", "vanilla-typescript-vanilla-ts", "new", "react-typescript-react-ts"],
-  "node": "14"
+  "node": "18"
```

Comment:
Node 20 will be LTS in October. I recommend using 20.

Reply:
Thanks. Let's use 18 unless it causes anything wrong.